extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -70,5 +70,25 @@ export const buildGtagConfig = (gtagId) => ({
         'repeat': false,
       },
     },
+    'storyShoppingProductEngagement': {
+      'on': 'story-shopping-product-engagement',
+      'vars': {
+        'event_name': 'custom',
+        'event_action': 'story_shopping_product_engagement',
```

Comment:
These events show up in the analytics dashboard after adding [`event_category`](https://developers.google.com/analytics/devguides/collection/gtagjs/events).

<img width="200" alt="Screen Shot 2022-03-24 at 2 20 34 PM" src="https://user-images.githubusercontent.com/3860311/159994047-6c15680b-e57d-45b8-9b78-2077bad80478.png">

It's awesome that you're debugging using dev tools and observing the `?collect` payloads. 
It's important to see how this looks from a users perspective. Setting up a analytics property is a good way to do that.

Reply:
I have added `event_cateory` to the analytics events.